File: chakeng/src/main/java/com/ssafy/chakeng/stats/dto/ReportStatistics.java
package com.ssafy.chakeng.stats.dto;

import java.util.List;

public record ReportStatistics(
        int totalReports,
        int completedReports,
        int pendingReports,
        int rejectedReports,
        int successRate,               // %
        int totalSuccessRate,          // 서비스 전체 대비 (필요시)
        List<ReportStatusStatistic> reportStatusStats
) {}

File: chakhaeng-android/app/src/main/java/com/sos/chakhaeng/domain/model/statistics/ReportStatistics.kt
package com.sos.chakhaeng.domain.model.statistics

data class ReportStatistics(
    val totalReports: Int,
    val completedReports: Int,
    val pendingReports: Int,
    val rejectedReports: Int,
    val successRate: Int, // 신고 성공률 (%)
    val totalSuccessRate: Int, // 서비스 이용자 전체 신고 성공률 (%)

    val reportStatusStats: List<ReportStatusStatistic>
)
